parser/parser_test.go

```diff
@@ -3700,6 +3701,16 @@ func TestDDL(t *testing.T) {
 		{"create resource group x ru_per_sec=3000 burstable", true, "CREATE RESOURCE GROUP `x` RU_PER_SEC = 3000 BURSTABLE"},
 		{"create resource group x burstable ru_per_sec=4000", true, "CREATE RESOURCE GROUP `x` BURSTABLE RU_PER_SEC = 4000"},
 		{"create resource group x ru_per_sec=20, priority=LOW, burstable", true, "CREATE RESOURCE GROUP `x` RU_PER_SEC = 20 PRIORITY = LOW BURSTABLE"},
+		{"create resource group x ru_per_sec=1000 QUERY LIMIT EXEC_ELAPSED_IN_SEC '10s' ACTION DRYRUN", true, "CREATE RESOURCE GROUP `x` RU_PER_SEC = 1000 QUERY LIMIT EXEC_ELAPSED_IN_SEC = '10s' ACTION DRYRUN"},
```

Comment:
IMO, I would like to make all runaway spec into one option, it looks more layered. like:
```
create resource group x ru_per_sec=1000 RUNAWAY_QUERY='{ LIMIT_EXEC_ELAPSED_IN_SEC="10s", ACTION=KILL}'
```